### PR TITLE
Support slack webhooks

### PR DESCRIPTION
**Pull request by rjulian**

### CODE REVIEW

File: modules/sns_email_subscription/main.tf [hunk 1]
```diff
@@ -19,6 +21,19 @@ resource "aws_cloudformation_stack" "sns_topic" {
     "Email" : {
       "Type" : "String",
       "Description" : "Email for SNS Notifications"
+    },
+    "SlackWebhookUrl": {
```
Comment (rot26) [inline modules/sns_email_subscription/main.tf:25]: @rjulian so is this a generic webhook, or does it carry a specific payload for slack?  If so, do we really want to be in the business of maintaining webhook payloads in slack?  I was under the impression this was going to be it's own custom lambda that was invoked by our SNS topic?

**Comment by rot26** [comment]
I was under the impression that reflex would handle events up to SNS.  everything after the SNS topic is a subscription handler.  I think it would be best to keep this separate from our lambda code

File: modules/sns_email_subscription/main.tf (hunk 1 above)
Comment (rjulian) [reply modules/sns_email_subscription/main.tf:25]: Reached out over chat. It is its own custom lambda  invoked by SNS.

Goal in the long run is to separate all output notification integrations.

Comment (rot26) [reply modules/sns_email_subscription/main.tf:25]: yup.  opened #92 to track this in the future.

**Pull request merged**